common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -779,7 +779,10 @@ def _load_items(self, course_entry, block_keys, depth=0, **kwargs):
             self._add_cache(course_entry.structure['_id'], runtime)
             self.cache_items(runtime, block_keys, course_entry.course_key, depth, lazy)
 
-        return [runtime.load_item(block_key, course_entry, **kwargs) for block_key in block_keys]
+        blocks = [runtime.load_item(block_key, course_entry, **kwargs) for block_key in block_keys]
+        for block in blocks:
+            block.course_version = course_entry.course_key.version_guid
```

Comment:
Ah.  I see.  My latest commit adds the field to old Mongo blocks as well.  Not sure if it's the best place to add it, but all paths do lead to it.